tests/semantics/manifest.ttl

```diff
@@ -1,12 +1,30 @@
-PREFIX rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
-PREFIX rdfs:   <http://www.w3.org/2000/01/rdf-schema#>
-PREFIX mf:     <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#>
-PREFIX test:   <http://www.w3.org/2001/sw/DataAccess/tests/test-dawg#>
-PREFIX xsd:    <http://www.w3.org/2001/XMLSchema#>
-PREFIX trs:    <https://w3c.github.io/rdf-star/tests/semantics#>
+## Distributed under both the "W3C Test Suite License" [1]
+## and the "W3C 3-clause BSD License".
+## [1] https://www.w3.org/Consortium/Legal/2008/04-testsuite-license
+## [2] https://www.w3.org/Consortium/Legal/2008/03-bsd-license
+
+@PREFIX rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
```

Comment:
I prefer `PREFIX`.

Is there a reason for `@` style?

But if we must use `@`, the `@prefix`.

Reply:
Yes, I fix that locally. I'll just use `PREFIX`